project-base/storefront/components/Basic/GoogleMap/GoogleMap.tsx

```diff
@@ -0,0 +1,142 @@
+import { GoogleMapMarker } from './GoogleMapMarker';
+import { useDomainConfig } from 'components/providers/DomainConfigProvider';
+import GoogleMapReact from 'google-map-react';
+import getConfig from 'next/config';
+import { useEffect, useMemo, useState } from 'react';
+import useSupercluster from 'use-supercluster';
+
+type GoogleMapMarker = {
+    identifier?: string;
+    uuid?: string;
+    locationLatitude: string | null;
+    locationLongitude: string | null;
+};
+
+type GoogleMapProps = {
+    lat?: string | null;
+    lng?: string | null;
```

Comment:
I would be a big fan of naming these as `latitude` and `longitude` everywhere.

Reply:
Adjusted.